### PR TITLE
Implement bottom tab UI

### PR DESCRIPTION
**Pull request by NguyenVanViet2004**
## Related issues
Resolved https://github.com/NguyenVanViet2004/FE-GlowUp/issues/6

## Summary
- Integrated bottom tabs navigation in `RootLayoutNav`
- Added new template for `home`, `booking`, `search`, and `profile`
- Added new translation keys and new color codes

### Steps:
1. Open `RootLayout` and locate the `RootLayoutNav` function.
2. **Comment out** the following line:

   ```jsx
   <Stack.Screen name="index" options={{ headerShown: false }} />
    ```
3. Launch the app and test the bottom tab navigation by switching between these tabs:
- Switch between the different tabs (Home, Booking, Profile, Search)
- Check that the title is correctly displayed according to the language setting
- Toggle between light and dark mode to ensure proper theming across all tabs

4. Verify the following:
- Titles are correctly displayed based on the selected language
- The app’s UI and colors properly adjust when switching between light and dark modes

## Screenshots
| Light mode | Dark mode |
| --- | --- |
|Simulator Screenshot - iPhone 15 Pro - 2024-09-22 at 15 52 39| Simulator Screenshot - iPhone 15 Pro - 2024-09-22 at 20 26 32|
|image|image|

### CODE REVIEW

File: tamagui.config.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { config as configBase } from '@tamagui/config/v3'
 import { createTamagui } from 'tamagui'
 
-import { fontConfig } from '~/fonts'
+import { fontConfig } from './fonts'
```
Comment (KMTus) [inline tamagui.config.ts:4]: Why not a relative path? 
import { fontConfig } from '~/fonts'

File: tamagui-web.css [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
...ht-9:41px;--f-lineHeight-10:59px;--f-lineHeight-11:69px;--f-lineHeight-12:76px;--f-lineHeight-13:87px;--f-lineHeight-14:109px;--f-lineHeight-15:133px;--f-lineHeight-16:155px;--f-lineHeight-true:23px;--f-weight-1:400;--f-weight-2:400;--f-weight-3:400;--f-weight-4:400;--f-weight-5:400;--f-weight-6:400;--f-weight-7:600;--f-weight-8:600;--f-weight-9:600;--f-weight-10:600;--f-weight-11:600;--f-weight-12:600;--f-weight-13:600;--f-weight-14:600;--f-weight-15:600;--f-weight-16:600;--f-weight-true:600;--f-letterSpacing-1:0px;--f-letterSpacing-2:0px;--f-letterSpacing-3:0px;--f-letterSpacing-4:0px;--f-letterSpacing-5:0px;--f-letterSpacing-6:0px;--f-letterSpacing-7:0px;--f-letterSpacing-8:0px;--f-letterSpacing-9:0px;--f-letterSpacing-10:0px;--f-letterSpacing-11:0px;--f-letterSpacing-12:0px;--f-letterSpacing-13:0px;--f-letterSpacing-14:0px;--f-letterSpacing-15:0px;--f-letterSpacing-16:0px;--f-letterSpacing-true:0px;--f-size-1:11px;--f-size-2:12px;--f-size-3:13px;--f-size-4:14px;--f-size-5:16px;--f-size-6:18px;--f-size-7:20px;--f-size-8:23px;--f-size-9:30px;--f-size-10:46px;--f-size-11:55px;--f-size-12:62px;--f-size-13:72px;--f-size-14:92px;--f-size-15:114px;--f-size-16:134px;--f-size-true:14px}
```
Comment (KMTus) [inline tamagui-web.css:6]: Is this CSS format having any problem?

File: tamagui.config.ts (hunk 1 above)
Comment (NguyenVanViet2004) [reply tamagui.config.ts:4]: When using a relative path here, an error may occur: "Could not resolve path."

File: tamagui-web.css (hunk 2 above)
Comment (NguyenVanViet2004) [reply tamagui-web.css:6]: No, it's good, but this file auto-changes when I work, so that's why I left it like that.

**Comment by truongtdph40209** [comment]
image

**Review by truongtdph40209 — APPROVED** [review]

**Comment by KMTus** [comment]
tukmph32467 review successfuly for issue 11: Implement Bottom Tab UI
image

**Review by KMTus — APPROVED** [review]

**Pull request merged**